beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/BeekeeperController.java

```diff
@@ -15,9 +15,23 @@
  */
 package com.expediagroup.beekeeper.api;
 
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+
 @RestController
 public class BeekeeperController {
-  
+
+  private HousekeepingEntityService housekeepingEntityService;
```

Comment:
Might be better to use the implementation class here because this service is specifically for `HousekeepingMetadata` so I don't think it should be generic.